Applications/Calendar/CalendarWidget.cpp

```diff
@@ -0,0 +1,169 @@
+#include "CalendarWidget.h"
+#include "AddEventDialog.h"
+#include <LibCore/DateTime.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Window.h>
+#include <LibGfx/Font.h>
+
+CalendarWidget::CalendarWidget()
+{
+    set_fill_with_background_color(true);
+    m_calendar = new Calendar(Calendar::get_year(), Calendar::get_month());
```

Comment:
Please don't use naked `new`. For a single-owner object, use `make<Calendar>` and store it in an `OwnPtr<Calendar>` :)